compiler/rustc_metadata/src/creader.rs

```diff
@@ -753,6 +757,7 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
                 hash,
                 extra_filename,
                 path_kind,
+                dep_kind == CrateDepKind::Stable,
```

Comment:
`CrateDepKind::Stable` never occurs in dep kinds coming from metadata, I think it's better for it to be a separate flag, not `CrateDepKind`.
Can put into into `CrateOrigin::Extern`, for example.